src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/grn.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <tuple>
+#include <string>
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+namespace ov {
+namespace test {
+typedef std::tuple<
+    ov::element::Type,
```

Comment:
description about each parameter will be useful

Reply:
done